test/cmocka/src/math/fft/fft.c

```diff
@@ -54,7 +54,7 @@ static void fft_real(struct comp_buffer *src, struct comp_buffer *dst, uint32_t
 	if (!outb)
```

Comment:
Need to add cmocka test case for 16 bit FFT and IFFT. Note also that FFT cmocka tests are not built and executed. Need to address that also.